runtime/src/main/java/com/redhat/ipaas/rest/Connections.java

```diff
@@ -30,58 +33,61 @@
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
-
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiParam;
+import java.util.Collection;
 
 @Path("/connections")
 @Api(value = "connections")
 public class Connections {
 
-	@Inject
-	private DataManager dataMgr;
-	
-	@GET
-	@Produces(MediaType.APPLICATION_JSON)
-	public Collection<Connection> list() {
-		return dataMgr.fetchAll(Connection.class);
-	}
-	
-	@GET
-	@Produces(MediaType.APPLICATION_JSON)
-	@Path(value="/{id}")
-	public Connection get(
-			@ApiParam(value = "id of the connection", required = true) @PathParam("id") String id) {
-		return dataMgr.fetch(Connection.class,id);
-	}
-	
-	@POST
-	@Produces(MediaType.APPLICATION_JSON)
-	@Consumes("application/json")
-	public String create(Connection connection) {
-		String id = dataMgr.create(connection);
-		return id;
-	}
-	
-	@PUT
-	@Path(value="/{id}")
-	@Consumes("application/json")
-	public void update(
-			@ApiParam(value = "id of the connection", required = true) @PathParam("id") String id, 
-			Connection connection) {
-		dataMgr.update(connection);
-		
-	}
-	
-	@DELETE
-	@Consumes("application/json")
-	@Path(value="/{id}")
-	public void delete(
-			@ApiParam(value = "id of the connection", required = true) @PathParam("id") String id) {
-		dataMgr.delete(Connection.class, id);
-		
-	}
-	
-	
-	
-}
\ No newline at end of file
+    @Inject
```

Comment:
I've set that up in `.editorconfig` already (https://github.com/redhat-ipaas/ipaas-api-java/blob/master/.editorconfig#L8-L9). See http://editorconfig.org if you don't know about editorconfig (it's got pretty wide editor support via plugins & is nice & simple).